src/main/resources/default/assets/common/core.js.pasta

```diff
@@ -258,19 +258,29 @@ sirius.getJSON = function (url, params) {
 
 /**@
  * Calls the given URL, posts the given params and retrieves the resulting JSON.
+ * <p>
+ * The parameters can either be specified as an object with key-value pairs (where the value is either a string or an
+ * array of strings), or directly as a URLSearchParams.
```

Comment:
Also eig kann man sowas in JSDoc dokumentieren.
zB `{Object|URLSearchParams}`. Dann wird das auch sauber angezeigt, wenn man auf der Methode die Quick Help anzeigt